src/main/java/com/github/fppt/jedismock/exception/WrongStreamKeyException.java

```diff
@@ -0,0 +1,7 @@
+package com.github.fppt.jedismock.exception;
```

Comment:
all the other exceptions in this package extend `RuntimeException`. Any purpose of this being the checked exception?

Reply:
I've made it a checked exception to ensure that when keys are being constructed from slices wrong data cases are checked. Otherwise it is a high probability of being missed.